Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/OutboundDocRetrieveStrategyBase.java

```diff
@@ -51,6 +52,7 @@
 public abstract class OutboundDocRetrieveStrategyBase implements OrchestrationStrategy {
 
     private static final Logger LOG = Logger.getLogger(OutboundDocRetrieveStrategyBase.class);
+    private final DocRetrieveAuditLogger docRetrieveAuditor = new DocRetrieveAuditLogger();
```

Comment:
Can this be static, or do we need one DocRetrieveAuditLogger instance per OutboundDocRetrieveStrategyBase instance?